src/software/Localization/main_SfM_Localization.cpp

```diff
@@ -22,6 +22,17 @@ using namespace openMVG::sfm;
 
 #include <cstdlib>
 
+// Naive function for finding the biggest common root dir from two paths
+std::string FindCommonRootDir(const std::string & dir1, const std::string & dir2)
```

Comment:
Just a note : I expected to find a function in stlplus that would do this but apparently it does not exist ? I might have missed it though !